mobile/build.gradle

```diff
@@ -140,7 +144,7 @@ dependencies {
     compile "info.metadude.android:typed-preferences:2.0.0"
     compile "org.jboss.aerogear:aerogear-android-authz:3.0.0"
     compile "org.jboss.aerogear:aerogear-android-pipe:3.1.0"
-    compile "org.jboss.aerogear:aerogear-android-push:3.0.0"
+    compile "org.jboss.aerogear:aerogear-android-push:4.0.0"
```

Comment:
Any reason to use `4.0.x` instead of `4.1.x`. If so please use the last bug fix version of `4.0.x` (`4.0.2`). 4.0.0 is buggy!